internal/view/table.go

```diff
@@ -210,6 +210,9 @@ func (t *Table) bindKeys() {
 		tcell.KeyCtrlW:         ui.NewKeyAction("Toggle Wide", t.toggleWideCmd, false),
 		ui.KeyShiftN:           ui.NewKeyAction("Sort Name", t.SortColCmd(nameCol, true), false),
 		ui.KeyShiftA:           ui.NewKeyAction("Sort Age", t.SortColCmd(ageCol, true), false),
+		ui.KeyQ:                ui.NewKeyAction("Reverse Sort Order", t.SortInvertCmd, false),
```

Comment:
I think `Q` is poor choice for the invert sort. This is on me since the keyspace is limited. 
Here is a thought... What if we just use the proposed PR and eliminate the canned sort keys. 
I think there are several benefits here:
1. Frees up the key space 
2. Allows us to use the defactor `R` or 'I'  to reverse on the selected sort column
3. This method becomes the standard k9s sort i.e solely 3 keys aka '<|>|R' and we can reclaim everything else

Do you think that would make sense?